conf.py

```diff
@@ -199,14 +199,14 @@
 html_css_files = [
     'css/algolia.css',
     'https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css',
-    "css/custom.css?10", # Increment the number at the end when the file changes to bust the cache.
```

Comment:
I think these cache busters are still used when deploying changes to the website CSS to get around long caching by the CDN/RTD. I don't think we can just remove them for the sake of EPUB, cc @YuriSizov

Reply:
Confirmed we still need and use these, so removing it won't work. Does this really break EPUB? It is a perfectly valid URL after all.